py3status/modules/sysdata.py

```diff
@@ -274,9 +292,20 @@ def _get_cputemp(self, zone, unit):
 
         return cpu_temp
 
+
     def sysdata(self):
         sys = {"max_used_percent": 0, "temp_unit": self.temp_unit}
 
+        cores = None
+        if self.init["cpu_freq_max"]:
+            cores = self._get_cpuinfo()
+            sys["cpu_freq_max"] = max(cores) / 1000  # convert to GHz
+
+        if self.init["cpu_freq_avg"]:
+            if cores is None:
+                cores = self._get_cpuinfo()
+            sys["cpu_freq_avg"] = (sum(cores) / len(cores)) / 1000
```

Comment:
Is it possible to get "Division by zero" exception and shall we guard somehow against it? Or it's unlikely that there will be no info in /proc/cpuinfo?

Reply:
It would mean that /proc isn't mounted, which is highly unusual. We'd be more likely to throw an IOError at `open("/proc/cpuinfo")`